citations-tool/tool/src/webapp/js/citationscript.js

```diff
@@ -671,11 +671,34 @@ function checkRequiredFields( alertMsg ) {
     return false;
   }
   
+  //If electronic citation check if at least one url is supplied
+  if($('#type_selector').val() == 'electronic'){
+      var showUrlAlert = $("input:text[name^='url_']" ).filter(function(){
```

Comment:
there is no existing class for the input text , do you want me to add a new class and then use that to select the element?